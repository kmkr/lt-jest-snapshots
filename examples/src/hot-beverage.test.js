import React from 'react';
import HotBeverage from './hot-beverage.js'
import renderer from 'react-test-renderer';

describe('HotBeverage', () => {
    it('should render correctly', () => {
        const tree = renderer.create(
            <HotBeverage/>
        ).toJSON();
        expect(tree).toMatchSnapshot();        
    });

    it('should render correctly when pouring in tea', () => {
        const component = renderer.create(<HotBeverage/>);
        const tree = component.toJSON();
        tree.children[1].props.onClick();
        expect(component.toJSON()).toMatchSnapshot();        
    });
});