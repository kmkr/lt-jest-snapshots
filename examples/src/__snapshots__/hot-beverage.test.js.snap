// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HotBeverage should render correctly 1`] = `
<div>
  <p>
    I am a HotBeverage and contains 
    coffee
  </p>
  <button
    onClick={[Function]}
  >
    Pour in tea
  </button>
</div>
`;

exports[`HotBeverage should render correctly with tea 1`] = `
<div>
  <p>
    I am a HotBeverage and contains 
    tea
  </p>
  <button
    onClick={[Function]}
  >
    Pour in tea
  </button>
</div>
`;
